virttest/utils_test/libvirt.py

```diff
@@ -3830,10 +3830,13 @@ def check_logfile(search_str, log_file, str_in_log=True,
     :param runner_on_target:  Remote runner
     :param ignore_status: True to return False on failure,
                           False to raise exception on failure
+    :param ignore_str: Ignore str
     :raise: test.fail when the result is not expected
     :return: True if check is successful, otherwise False
     """
     cmd = "grep -E '%s' %s" % (search_str, log_file)
+    if len(ignore_str) != 0:
+        cmd += f" -v {ignore_str}"
```

Comment:
Adding '-v ignore_str' after 'log_file' will cause a failure. Are you going to add `cmd += f"| grep -v {ignore_str}"`?